lib/app/features/nostr/providers/nostr_notifier.c.dart

```diff
@@ -193,3 +242,15 @@ class NostrNotifier extends _$NostrNotifier {
     return entity;
   }
 }
+
+extension on List<UserRelay> {
+  List<UserRelay> avoiding(List<String> dislikedRelaysUrls) {
```

Comment:
Maybe add some comment explaining what `dislike relays` means to avoid ambiguity.